index.js

```diff
@@ -0,0 +1,239 @@
+  password: config.get("lbrycrd.password"),
+  port: config.get("lbrycrd.port")
+});
+const logger = winston.createLogger({
+  level: "info",
+  format: winston.format.json(),
+  transports: [
+    new winston.transports.DailyRotateFile({
+      filename: "tipbot-%DATE%.log",
+      dirname: "./logs",
+      datePattern: "YYYY-MM-DD-HH",
+      zippedArchive: true,
+      maxSize: "20m",
+      maxFiles: "14d"
+    }),
+    new winston.transports.Console({
+      format: winston.format.simple(),
+      level: "debug"
+    })
+  ]
+});
+
+const T = new Twit({
+  consumer_key: config.get("twitter.consumer_key"),
+  consumer_secret: config.get("twitter.consumer_secret"),
+  access_token: config.get("twitter.access_token"),
+  access_token_secret: config.get("twitter.access_token_secret"),
+  timeout_ms: 60 * 1000, // optional HTTP request timeout to apply to all requests.
+  strictSSL: true // optional - requires SSL certificates to be valid.
+});
+
+const stream = T.stream("statuses/filter", { track: config.get("bot.handle") });
+logger.info("Started LBRY twitter tipbot.");
+
+stream.on("tweet", function(tweet) {
+  if(tweet.user.screen_name === config.get("bot.handle").substring(1)) return;
+  let msg = checkTrunc(tweet);
+  msg = msg.slice(msg.indexOf(config.get("bot.handle"))).split(" ");
+  checkTweet(tweet, msg);
+});
+
+function checkTweet(tweet, msg) {
+  switch (msg[1]) {
+    case "help":
+      doHelp(tweet, msg);
+      break;
+    case "balance":
+      doBalance(tweet, msg);
+      break;
+    case "deposit":
+      doDeposit(tweet, msg);
+      break;
+    case "withdraw":
+      doWithdraw(tweet, msg);
+      break;
+    case "tip":
+      doTip(tweet, msg);
+      break;
+    case "terms":
+      doTerms(tweet, msg);
+      break;
+    case "lbryian":
+      logger.info("Got a command with the old format, handling it...");
+      checkTweet(tweet, msg.splice(1));
+      break;
+  }
+}
+
+async function doHelp(tweet, msg) {
+  try {
+    let post = await T.post("statuses/update", {
+      status:
+        `@${tweet.user.screen_name} `+
+        `All commands should be called with ${config.get("bot.handle")} + subcommand \n` +
+        "help - Shows this command. \n" +
+        "balance - Get your balance. \n" +
+        "deposit - Get address for your deposits. \n" +
+        "withdraw ADDRESS AMOUNT - Withdraw AMOUNT credits to ADDRESS. \n" +
+        "tip USER AMOUNT - Tip USER AMOUNT.",
+      in_reply_to_status_id: tweet.id_str
+    });
+    logger.info(
+      `Sent help to ${tweet.user.screen_name}, tweet id: ${tweet.id_str}`
+    );
+  } catch (e) {
+    logger.error(e);
+  }
+}
+async function doTerms(tweet, msg){
+// ADD terms
+  await T.post("statuses/update", {
+    status:
+    `@${tweet.user.screen_name} `+
+    "There are no fees to use this bot except the automatic daemon fee. \n"+
+    "In no event shall LBRY Inc be responsible in the event of lost, stolen or misdirected funds.",
```

Comment:
My English isn't exceptional but I think the sentence should be changed to:

`Under no circumstances shall LBRY Inc. be held responsible for lost, stolen or misdirected funds.`

Reply:
Will fix!